engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -21,41 +21,43 @@
 import org.terasology.config.RenderingConfig;
 import org.terasology.context.Context;
 import org.terasology.rendering.opengl.AbstractFBOsManager;
-import org.terasology.rendering.opengl.DefaultDynamicFBOs;
-import static org.terasology.rendering.opengl.DefaultDynamicFBOs.FINAL;
-import static org.terasology.rendering.opengl.DefaultDynamicFBOs.READ_ONLY_GBUFFER;
-import static org.terasology.rendering.opengl.DefaultDynamicFBOs.WRITE_ONLY_GBUFFER;
+import static org.terasology.rendering.opengl.ScalingFactors.FULL_SCALE;
+
 import org.terasology.rendering.opengl.FBO;
 import org.terasology.rendering.opengl.FBOConfig;
 import org.terasology.rendering.opengl.ScreenGrabber;
+import org.terasology.rendering.world.WorldRenderer;
+import org.terasology.rendering.world.WorldRendererImpl;
 
 /**
  * TODO: Add javadocs
  * TODO: Better naming
  */
 public class DisplayResolutionDependentFBOs extends AbstractFBOsManager {
+    public static final ResourceUrn SCENE_OPAQUE = new ResourceUrn("engine:sceneOpaque");
+    public static final ResourceUrn SCENE_OPAQUE_PING_PONG = new ResourceUrn("engine:sceneOpaquePingPong");
+    public static final ResourceUrn SCENE_FINAL = new ResourceUrn("engine:sceneFinal");
```

Comment:
I would recommend against naming it as simply FINAL. The reason is, this variable is used in a lot of places in the code, and FINAL will get pretty confusing. FINAL what? FINAL_BUFFER, READONLY_GBUFFER and WRITEONLY_GBUFFER make more sense, IMHO.